src/components/GameView/GameDialogs.vue

```diff
@@ -99,57 +89,57 @@ export default {
     // TODO: Refactor and combine usage in GameView.vue
     mustReauthenticate: {
       get() {
-        return this.$store.state.auth.mustReauthenticate;
+        return this.authStore.mustReauthenticate;
       },
       set(val) {
-        this.$store.commit('setMustReauthenticate', val);
+        this.authStore.setMustReauthenticate(val);
```

Comment:
It probably makes sense to remove any former mutation functions that just set a single variable and replace them with inline assignments
```suggestion
        this.authStore.mustReauthenticate = val;
```